README.md

```diff
@@ -78,9 +78,12 @@ Below are instructions for setting up project development.
 1. Run a development server with `npm run dev`
     - View the demos by accessing `<dev server url>`/demos/index.html
     - This will also build the project.
-1. Run tests with `npm run test`
+1. Run tests with `npm run build:tests && npm run test`
 1. Run benchmarks with `npm run build:tests && npm run test:bench`
 
+*Note:* You can debug specific tests by running `npm run test:debug -- --packages mwc-button,mwc-fab`
+or you may use wildcards such as `npm run test:debug -- --packages mwc-icon-button*`
```

Comment:
Maybe also add that you can run on just one browser with e.g. `--browsers=ChromeHeadless` (which is what I usually do when iterating on a test, since it's the fastest).